docs/faq.md

```diff
@@ -51,6 +51,29 @@ following:
 }
 ```
 
+The same technique works for extracting element attributes into variables.
+Here's an example of storing an element's text attribute.
+
+```swift
+// Swift
+/*
+ *  Example Usage:
+ *
+ *  var textValue:String = ""
+ *
+ *  domainField.performAction(grey_typeText("hello.there"))
+ *             .performAction(grey_getText(&textValue))
+ */
+func grey_getText(inout text: String) -> GREYActionBlock {
```

Comment:
Cool, we should probably add that to the contributing.md file. I pushed a new update.